Src/Private/Get-AbrAppVolPackage.ps1

```diff
@@ -0,0 +1,296 @@
+                                                                        'Path' = $StorageLocation.path
+                                                                        'Host' = $StorageLocation.machine_manager_host
+                                                                        'File Status' = $StorageLocation.reachable
+                                                                        'Created' = $StorageLocation.created_At_Human
+                                                                    }
+                                                                    $OutObj += [pscustomobject](ConvertTo-HashToYN $inObj)
+                                                                }
+                                                                catch {
+                                                                    Write-PscriboMessage -IsWarning $_.Exception.Message
+                                                                }
+                                                            }
+
+                                                            $TableParams = @{
+                                                                Name = "Storage Location for - $($AppPackage.Name)"
+                                                                List = $false
+                                                                ColumnWidths = 20, 30, 20, 15, 15
+                                                            }
+                                                            if ($Report.ShowTableCaptions) {
+                                                                $TableParams['Caption'] = "- $($TableParams.Name)"
+                                                            }
+                                                            $OutObj | Sort-Object -Property 'Location' |  Table @TableParams
+                                                        }
+                                                    }
+                                                }  catch {
+                                                    Write-PscriboMessage -IsWarning $_.Exception.Message
+                                                }
+                                                try {
+                                                    if ($PSVersionTable.PSEdition -eq 'Core') {
+                                                        $AppLinks = Invoke-RestMethod -SkipCertificateCheck -WebSession $SourceServerSession -Method Get -Uri "https://$AppVolServer/app_volumes/app_packages/$($AppPackage.id)/app_links?"
+                                                    } else {$AppLinks = Invoke-RestMethod -WebSession $SourceServerSession -Method Get -Uri "https://$AppVolServer/app_volumes/app_packages/$($AppPackage.id)/app_links?"}
+
+                                                    if ($AppLinks) {
```

Comment:
```suggestion
                                                    if ($AppLinks.data) {
```

```
VERBOSE: [ 20:35:56:335 ] [ Document ] - Processing section 'Application links for - Google Chrome 88' started.
VERBOSE: [ 20:35:56:336 ] [ Document ] - Processing table 'Application links for - Google Chrome 88'.
WARNING: [ 20:35:56:342 ] [ Document ] - Cannot bind argument to parameter 'Rows' because it is an empty collection.
```

If the package has no application links, the error "Cannot bind argument to parameter 'Rows' because it is an empty collection" is displayed.

Reply:
Changed if ($AppLinks) { to 
if ($AppLinks.data) {